addon/helpers/cloudinary-url.js

```diff
@@ -32,69 +21,13 @@ export default Ember.Helper.extend({
    * @param  {String[]}        hash.transforms
    * @return {String}                               URL for image
    */
-  compute([publicId] = [], hash = {}) {
+  compute(publicId, hash) {
```

Comment:
The `publicId` can returned to be in brackets (`[publicId]`). Since we only need the first parameter of the array. And the first parameter of `Ember.Helper#compute` is always an array.